bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -126,5 +137,71 @@
 		<description>A detailed message describing an alert condition with the chamber</description>
 		<state readOnly="true"/>
 	</channel-type>
+	<channel-type id="firstNameType">
+		<item-type>String</item-type>
+		<label>First Name</label>
+		<description>The first name of the sleeper</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="sleepGoalMinutesType">
+		<item-type>Number</item-type>
```

Comment:
Could this be `Number:Time` for UoM support (and updated as `QuantityType`)?

Reply:
Yes